src/components/standalone/ssh/SshKeys.vue

```diff
@@ -198,6 +198,7 @@ function handleCloseDeleteModal() {
           :invalid-message="t(validationErrors.getFirstI18nKeyFor('key'))"
           :label="t('standalone.ssh.ssh_keys.add_new_ssh_key.label')"
           :placeholder="t('standalone.ssh.ssh_keys.add_new_ssh_key.placeholder')"
+          required
```

Comment:
This should be validated by showing `invalid-message` for consistency, but it's not a mandatory change